doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc

```diff
@@ -0,0 +1,48 @@
+[id='secret-adminuser-create-proc']
+:INCLUDECENTRAL!:
+ifeval::["{context}"=="openshift-authoring"]
+:INCLUDECENTRAL: {CENTRAL}
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:INCLUDECENTRAL: {CENTRAL}
+endif::[]
+ifeval::["{context}"=="openshift-freeform"]
+:INCLUDECENTRAL: {CENTRAL} Monitoring
+endif::[]
+ifeval::["{context}"=="openshift-managed"]
+:INCLUDECENTRAL: {CENTRAL} Monitoring
+endif::[]
+//immutable handled separately, so do not define INCLUDECENTRAL for it
+
+
+= Creating the secret for the administrative user
+
+You must create a secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template. 
```

Comment:
```suggestion
You must create a generic secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template.
```
Maybe it should be mentioned that the secret keys must be KIE_ADMIN_USER and KIE_ADMIN_PWD